Eatery/Onboarding/Controllers/OnboardingLoginViewController.swift

```diff
@@ -55,18 +56,33 @@ class OnboardingLoginViewController: OnboardingViewController {
         setUpErrorView()
         setUpNetidViews()
         setUpPasswordViews()
+        setUpButton()
         setUpSkipButton()
+
+        stackView.layoutIfNeeded()
+        contentView.snp.makeConstraints { make in
+            make.width.equalToSuperview()
+            // 52.5 is the height of the errorView
+            make.height.equalTo(stackView.frame.height + 52.5)
```

Comment:
Once more about the `make.height.equalTo(stackView.height)`

Reply:
If we are adding  52.5 we have to specify .frame.height

Reply:
Actually we do not even need to add the 52.5